lib/internal/util/comparisons.js

```diff
@@ -387,12 +387,10 @@ function findLooseMatchingPrimitives(prim) {
     case 'symbol':
       return false;
     case 'string':
-      const number = +prim;
+      prim = +prim;
```

Comment:
Not a huge fan of this change (makes it harder to follow IMO) but still LGTM on the PR.

Reply:
Shall I change it back?

Reply:
No strong feelings - I'm just not a fan of this sort of assignment since it takes another extra step to follow - but you can absolutely land as is if you want.